# num_guess_try_catch.py